node.js

```diff
@@ -1,25 +0,0 @@
-// These are rules that apply to node.js scripts.  We use a recent
-// version of node.js that provides most, but not all, ES6 constructs.
-// So we inherit from the javascript-file index.js, which includes
-// ES6.  (It also includes a bunch of react stuff, which is not
-// relevant for node.js scripts but also not harmful.)
-//
-// The major differences are that console statements are welcome
-// in node scripts!  Also, the list of globals is different.
-// TODO(csilvers): turn off es6 lint-rules for features that are
-// not supported by node yet.
-//
-// In general, you should be adjusting index.js or shared.js
-// instead of this file.  Only adjust this file if you are making
-// a change that accounts for some quirks in node.js in particular.
-
-module.exports = {
-    rules: {
-        "no-console": 0,
```

Comment:
It seems like we may still want configs that work like this, right?

Reply:
I think we'll want to disable this rule on per-repo basis.  In the case of webapp we can enable it for the `javascript` folder and disable it for all other folders.